UPGRADE-5.3.md

```diff
@@ -43,6 +43,8 @@ FrameworkBundle
  * Deprecate the `session` service and the `SessionInterface` alias, use the `\Symfony\Component\HttpFoundation\Request::getSession()` or the new `\Symfony\Component\HttpFoundation\RequestStack::getSession()` methods instead
  * Deprecate the `KernelTestCase::$container` property, use `KernelTestCase::getContainer()` instead
  * Rename the container parameter `profiler_listener.only_master_requests` to `profiler_listener.only_main_requests`
+ * Deprecate the public `workflow.abstract` and `state_machine.abstract` abstract definitions to private
+ * Deprecate creating workflow services from configuration as public to private
```

Comment:
I need help for that sentence.

Reply:
`  * Deprecate registering workflow services as public`